<launch>
    <arg name="joy_dev" default="/dev/input/js0" />

    <node pkg="joy" type="joy_node" name="joy_node">
        <param name="dev" value="$(arg joy_dev)" />
        <param name="deadzone" value="0.0" />
        <param name="autorepeat_rate" value="0.0" />
    </node>

    <rosparam file="$(find ros_pca9685_drive)/config/joystick_xbox.yaml" command="load" />
    <node pkg="joy_teleop" type="joy_teleop.py" name="joy_teleop" />

    <node pkg="ros_pca9685_drive" type="pca9685_node.py" name="pca9685_drive" output="screen">
        <rosparam file="$(find ros_pca9685_drive)/config/pca9685.yaml" command="load" />
    </node>

</launch>
